react/features/share-room/middleware.js

```diff
@@ -40,28 +40,30 @@ MiddlewareRegistry.register(store => next => action => {
  */
 function _shareRoom(
         roomURL: string, includeDialInfo: boolean, dispatch: Function) {
-    const message = getShareInfoText(roomURL, includeDialInfo);
-    const title = `${getName()} Conference`;
-    const onFulfilled
-        = (shared: boolean) => dispatch(endShareRoom(roomURL, shared));
+    dispatch(getShareInfoText(roomURL, includeDialInfo))
```

Comment:
I haven't tested it, I will re-test it. On the welcome page, it will not indicate any change in the UI ... so maybe it will be confusing, but we do not have any notifications there I think, so not sure how to handle it. Totally not sure about mobile though.